roles/network/tasks/detected_network.yml

```
@@ -313,6 +318,10 @@
       value: "{{ can_be_ap }}"
     - option: host_country_code_found
       value: "{{ host_country_code }}"
+    - option: firmware_option_1
+      value:  "{{ rpi3bplus_rpi4_wifi_firmware }}"
+    - option: firmware_option_2
+      value:  "{{ rpizerow_rpi3_wifi_firmware }}"
```

Even prior to renaming these 2 variables ([very likely in coming months or weeks](https://github.com/iiab/iiab/issues/3881#issuecomment-2591534339)) I'd suggest re-ordering & clarifying here, to get out in front making this iiab.ini stuff cleaner:

```suggestion
    - option: wifi_firmware_43430
      value: "{{ rpizerow_rpi3_wifi_firmware }}"
    - option: wifi_firmware_43455
      value: "{{ rpi3bplus_rpi4_wifi_firmware }}"
```